src/services/CtypeRepository.ts

```diff
@@ -45,7 +45,7 @@ class CTypeRepository {
       })
   }
 
-  public static async register(cType: ICType): Promise<Response> {
+  public static async register(cType: CTypeMetadata): Promise<Response> {
```

Comment:
I had to change the type to match the input. Unfortunately, this is the problem I am facing with the BScType within the dev tools. It takes the sdk.Ctype, not the newly created CTypeMetadata.